.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,62 @@
+name: Test Emscripten/Pyodide build
+
+on:
+  push:
+    branches:
+      - emscripten-ci
+  pull_request:
+    branches:
+      - main
+      - maintenance/**
+
+jobs:
+  build-wasm-emscripten:
+    # run this for now on all CI for now in case something starts breaking it
+    # if: "success() && (startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/main')"
+    # needs: [test, lint]
+    runs-on: ubuntu-latest
```

Comment:
Maybe `ubuntu-22.04` as I'm not sure this is forward compatible?

Reply:
Most of the other ci jobs run in ubunti-latest though there are a few that are version pinned. We can leave it up to the numpy maintainers. @mattip @seberg opinions?

Reply:
I don't care.  We will probably ping you anyway if the job starts failing ;).

Reply:
Agreed, it's fine either way.